vision/lidar/record_data.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+""" Record data from Ouster OS-1-32-U LiDAR sensor to pcap file."""
+
+import os
+import sys
+import argparse
+import time
+from contextlib import closing
+from datetime import datetime
+from more_itertools import time_limited
+
+from ouster import client, pcap
+
+HOSTNAME = "os-122229001687.local"
+LIDAR_PORT = 7502
+IMU_PORT = 7503
+
+
+def record_pcap(
+    hostname: str,
+    lidar_port: int = 7502,
+    imu_port: int = 7503,
```

Comment:
Should the parameters for this function be defaulting to the constants you have defined?

Reply:
This function was honestly directly copy-pasted from their source code and modified a bit for customizability. So I kinda forgot to organize it with type hints and stuff. I'll try to clean it up.